source/uwp/SharedVisualizer/ViewModel/HostConfigEditorViewModel.cs

```diff
@@ -73,5 +75,26 @@ public static async Task<HostConfigEditorViewModel> LoadAsync(MainPageViewModel
                 return new HostConfigEditorViewModel(mainPageViewModel);
             }
         }
+
+        protected static string PickHostConfigFile()
+        {
+            AccessibilitySettings accessibilitySettings = new AccessibilitySettings();
```

Comment:
While you're doing this, it's probably worth wiring up a [`HighContrastChanged` event handler](https://learn.microsoft.com/en-us/uwp/api/windows.ui.viewmanagement.accessibilitysettings.highcontrastchanged?view=winrt-22621) so that user contrast theme changes are automatically applied.